### ISSUE TITLE
All of the card reports are broken

### ISSUE THREAD

**Issue by ftrotter** [CareSet/Zermelo#123]
All of the card reports are broken on lore.ft1.us

https://lore.ft1.us/ZermeloCard/CardSearchUsingCard
https://lore.ft1.us/ZermeloCard/CardSearchUsingImgGrid
https://lore.ft1.us/ZermeloCard/cardMostSets

Is this fixed with the upgrade to Ubuntu 20.x ?
Let me know if this is working on your test servers..

-ft

**Comment by kchapple** [comment]
@seanccsmith this has been fixed, and was related to #122 

[master fea4a7c] Fixed sort order parameter bug, thro exception when parameters are wrong format (#123) (#122)

**Comment by seanccsmith** [comment]
The classic card viewer report is now working, but the card search reports remain broken.

**Comment by kchapple** [comment]
@seanccsmith @ftrotter I tracked down a clue why card search is not working on lore server. I ran the following query on lore.ft1.us and on my local. My local returns results with text in each row. On ft1 returns empty strings for each row. The column for_fulltext_search gets populated in app/Console/Commands/PostSync.php so it's possible that we either need to sync the server's data, or the sync is broken.

`select for_fulltext_search from cardface LIMIT 100;`
